core/templates/pages/exploration-editor-page/services/exploration-tags.service.spec.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the ExplorationTagsService.
+ */
+
+// TODO(#7222): Remove the following block of unnnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+require(
+  'pages/exploration-editor-page/services/exploration-tags.service.ts');
+
+describe('Exploration Tags Service', function() {
+  let ets = null;
+
+  beforeEach(function() {
+    angular.mock.module('oppia');
+    angular.mock.module(function($provide) {
+      let ugs = new UpgradedServices();
+      for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+        $provide.value(key, value);
+      }
+    });
```

Comment:
```suggestion
beforeEach(angular.mock.module('oppia', function($provide) {
    var ugs = new UpgradedServices();
    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
      $provide.value(key, value);
    }
  }));
```

Reply:
it is showing an error, do you have any file like this so that I can have a look.

Reply:
Can you please describe the error you're getting? You can take a look on `training-modal.service.spec.ts` for more details

Reply:
sorry I replied late. I did some changes based on your suggestion PTAL.

Reply:
No worries, thanks @safwan20